Invocation_TimeoutTest.async_whenCallTimeout_thenOperationTimeoutException & sync_whenCallTimeout_thenOperationTimeoutException

```
DummyOperation invocation failed to complete due to operation-heartbeat-timeout. 
Total elapsed time: 10665 ms. 
Last heartbeat: never. Invocation{op=com.hazelcast.spi.impl.operationservice.impl.DummyOperation{serviceName='null', identityHash=1593789295, partitionId=1, replicaIndex=0, callId=0, invocationTime=1460335510642 (Mon Apr 11 03:45:10 EEST 2016), waitTimeout=-1, callTimeout=5000}, tryCount=250, tryPauseMillis=500, invokeCount=1, callTimeoutMillis=5000, firstInvocationTimeMs=1460335510642, firstInvocationTime='Mon Apr 11 03:45:10 EEST 2016', lastHeartbeatMillis=0, lastHeartbeatTime='Thu Jan 01 02:00:00 EET 1970', target=Address[127.0.0.1]:7970, pendingResponse={VOID}, backupsExpected=0, backupsCompleted=0, connection=MockConnection{localEndpoint=Address[127.0.0.1]:7969, remoteEndpoint=Address[127.0.0.1]:7970}}
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-ZuluJDK8/com.hazelcast$hazelcast/171/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_TimeoutTest/async_whenCallTimeout_thenOperationTimeoutException/

**sync_whenCallTimeout_thenOperationTimeoutException**
```
java.lang.AssertionError: DummyOperation invocation failed to complete due to operation-heartbeat-timeout. Current time: 2016-06-05 05:41:54.230. Total elapsed time: 10035 ms. Last operation heartbeat: never. Last operation heartbeat from member: 2016-06-05 05:41:53.951. Invocation{op=com.hazelcast.spi.impl.operationservice.impl.DummyOperation{serviceName='null', identityHash=1386057644, partitionId=1, replicaIndex=0, callId=0, invocationTime=1465094504195 (2016-06-05 05:41:44.195), waitTimeout=-1, callTimeout=5000}, tryCount=250, tryPauseMillis=500, invokeCount=1, callTimeoutMillis=5000, firstInvocationTimeMs=1465094504195, firstInvocationTime='2016-06-05 05:41:44.195', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 02:00:00.000', target=[127.0.0.1]:11922, pendingResponse={VOID}, backupsAcksExpected=0, backupsAcksReceived=0, connection=MockConnection{localEndpoint=[127.0.0.1]:11921, remoteEndpoint=[127.0.0.1]:11922}}
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_TimeoutTest.sync_whenCallTimeout_thenOperationTimeoutException(Invocation_TimeoutTest.java:321)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-OracleJDK1.6/com.hazelcast$hazelcast/931/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_TimeoutTest/sync_whenCallTimeout_thenOperationTimeoutException/

**sync_whenCallTimeout_thenOperationTimeoutException**
```
java.lang.AssertionError: DummyOperation invocation failed to complete due to operation-heartbeat-timeout. Current time: 2016-06-14 06:15:12.026. Total elapsed time: 10693 ms. Last operation heartbeat: never. Last operation heartbeat from member: 2016-06-14 06:15:11.025. Invocation{op=com.hazelcast.spi.impl.operationservice.impl.DummyOperation{serviceName='null', identityHash=519580774, partitionId=3, replicaIndex=0, callId=0, invocationTime=1465874101333 (2016-06-14 06:15:01.333), waitTimeout=-1, callTimeout=5000}, tryCount=250, tryPauseMillis=500, invokeCount=1, callTimeoutMillis=5000, firstInvocationTimeMs=1465874101333, firstInvocationTime='2016-06-14 06:15:01.333', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 02:00:00.000', target=[127.0.0.1]:13293, pendingResponse={VOID}, backupsAcksExpected=0, backupsAcksReceived=0, connection=MockConnection{localEndpoint=[127.0.0.1]:13292, remoteEndpoint=[127.0.0.1]:13293}}
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_TimeoutTest.sync_whenCallTimeout_thenOperationTimeoutException(Invocation_TimeoutTest.java:321)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-OracleJDK1.7/com.hazelcast$hazelcast/512/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_TimeoutTest/sync_whenCallTimeout_thenOperationTimeoutException/

**async_whenCallTimeout_thenOperationTimeoutException**
```
java.lang.AssertionError: DummyOperation invocation failed to complete due to operation-heartbeat-timeout. Current time: 2016-06-15 07:08:58.955. Total elapsed time: 10338 ms. Last operation heartbeat: never. Last operation heartbeat from member: 2016-06-15 07:08:58.342. Invocation{op=com.hazelcast.spi.impl.operationservice.impl.DummyOperation{serviceName='null', identityHash=1503447421, partitionId=0, replicaIndex=0, callId=0, invocationTime=1465963728617 (2016-06-15 07:08:48.617), waitTimeout=-1, callTimeout=5000}, tryCount=250, tryPauseMillis=500, invokeCount=1, callTimeoutMillis=5000, firstInvocationTimeMs=1465963728617, firstInvocationTime='2016-06-15 07:08:48.617', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 02:00:00.000', target=[127.0.0.1]:5072, pendingResponse={VOID}, backupsAcksExpected=0, backupsAcksReceived=0, connection=MockConnection{localEndpoint=[127.0.0.1]:5071, remoteEndpoint=[127.0.0.1]:5072}}
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_TimeoutTest$2.run(Invocation_TimeoutTest.java:64)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:884)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:898)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_TimeoutTest.assertEventuallyFailsWithCallTimeout(Invocation_TimeoutTest.java:56)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_TimeoutTest.async_whenCallTimeout_thenOperationTimeoutException(Invocation_TimeoutTest.java:344)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x/com.hazelcast$hazelcast/5256/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_TimeoutTest/async_whenCallTimeout_thenOperationTimeoutException/

--- a/hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/Invocation_TimeoutTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/Invocation_TimeoutTest.java
@@ -298,7 +298,7 @@ public class Invocation_TimeoutTest extends HazelcastTestSupport {
 
     @Test
     public void sync_whenCallTimeout_thenOperationTimeoutException() throws Exception {
-        long callTimeoutMs = 5000;
+        long callTimeoutMs = 10000;
         Config config = new Config().setProperty(GroupProperty.OPERATION_CALL_TIMEOUT_MILLIS.getName(), "" + callTimeoutMs);
         TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
         HazelcastInstance local = factory.newHazelcastInstance(config);
@@ -308,9 +308,9 @@ public class Invocation_TimeoutTest extends HazelcastTestSupport {
         OperationService opService = getOperationService(local);
 
         int partitionId = getPartitionId(remote);
-        opService.invokeOnPartition(null, new SlowOperation(callTimeoutMs * 2), partitionId);
+        opService.invokeOnPartition(new SlowOperation(callTimeoutMs * 2).setPartitionId(partitionId));
 
-        Future f = opService.invokeOnPartition(null, new DummyOperation(), partitionId);
+        Future f = opService.invokeOnPartition(new DummyOperation().setPartitionId(partitionId));
 
         try {
             f.get(3 * callTimeoutMs, MILLISECONDS);
@@ -324,7 +324,7 @@ public class Invocation_TimeoutTest extends HazelcastTestSupport {
 
     @Test
     public void async_whenCallTimeout_thenOperationTimeoutException() throws Exception {
-        long callTimeoutMs = 5000;
+        long callTimeoutMs = 10000;
         Config config = new Config().setProperty(GroupProperty.OPERATION_CALL_TIMEOUT_MILLIS.getName(), "" + callTimeoutMs);
         TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
         HazelcastInstance local = factory.newHazelcastInstance(config);
@@ -334,9 +334,9 @@ public class Invocation_TimeoutTest extends HazelcastTestSupport {
         OperationService opService = getOperationService(local);
 
         int partitionId = getPartitionId(remote);
-        opService.invokeOnPartition(null, new SlowOperation(callTimeoutMs * 2), partitionId);
+        opService.invokeOnPartition(new SlowOperation(callTimeoutMs * 2).setPartitionId(partitionId));
 
-        ICompletableFuture f = opService.invokeOnPartition(null, new DummyOperation(), partitionId);
+        ICompletableFuture f = opService.invokeOnPartition(new DummyOperation().setPartitionId(partitionId));
 
         ExecutionCallback callback = mock(ExecutionCallback.class);
         f.andThen(callback);

